lib/tasks/install-blueprint.js

```diff
@@ -80,6 +113,30 @@ class InstallBlueprintTask extends Task {
     return execa('npm', ['install'], { cwd });
   }
 
+  _npmInstallModule(module, cwd) {
+    logger.info(`Running "npm install ${module}" in "${cwd}" ...`);
+    return execa('npm', ['install', module], { cwd })
```

Comment:
gotcha, seems good, can you either confirm this is encompassed by an existing issue or create a new issue (so that we don't loose track)?